spring-cloud-azure-config/src/main/java/com/microsoft/azure/spring/cloud/config/AzureConfigPropertySourceLocator.java

```diff
@@ -99,4 +101,13 @@ private AzureConfigPropertySource create(String context) {
 
         return propertySource;
     }
+
+    @Override
+    public int getOrder() {
+        return order;
+    }
+
+    public void setOrder(int order) {
```

Comment:
Is this method just for testing?

Reply:
It's not, if user needs to configure different order, they have to use this method and create a different bean, same as the test configuration.

Reply:
Updated to configure with properties.